src/main/java/org/openmrs/uitestframework/page/Page.java

```diff
@@ -362,7 +362,7 @@ public Boolean containsText(String text) {
 
     public List<String> getValidationErrors() {
         List<String> validationErrors = new ArrayList<String>();
-        for (WebElement webElement : driver.findElements(By.className("error"))) {
+        for (WebElement webElement : driver.findElements(By.className("field-error"))) {
```

Comment:
where does this get renamed in the reference application to make `field-error` usable across all places where `error` is used

Reply:
Thanks @k-joseph  ,  i have been looking into  exactly where this has been applied,For all the other classes with in distribution,we only one have single place where we are implementing getInvalidPhoneNumberNotification  here https://github.com/openmrs/openmrs-distro-referenceapplication/blob/a3a87b30a694b448ea9776936c765648a0b5a6d5/ui-tests/src/test/java/org/openmrs/reference/page/RegistrationEditSectionPage.java#L117 . Since getValidationErrors method is a generic concept ,so it can be used across the entire platform, it allows itself to be used in different ways

On this hand its being used to validate phone Number at registration when you enter a non value or string which it will throw an error message and that error message is the one we are inspecting to get the right field element to use here

Reply:
> On the the other hand it being used to validate phone Number at registration when you enter a non value or string where it will throws an error message and that error message is the one we are inspecting to get the right field element to use here

You may need to revisit on my comments I attached on the ticket [here](https://issues.openmrs.org/browse/RATEST-165)

Reply:
Do you require more explanation morethan that, is this clarification lacking something

Reply:
I rather mean https://github.com/openmrs/openmrs-distro-referenceapplication/search?q=getValidationErrors

Doesn't this element change break any of those. You can such through the reference application to be sure the element `error` was renamed to `field-error` since it used to work right

Reply:
Thanks @k-joseph  @kdaud for The review, As you can see here  with in this commit, https://github.com/openmrs/openmrs-contrib-uitestframework/commit/e35fe557d12a229c6a389d61534947c0026ebe9f#diff-de35e2859130a4c8552c1bc12624bc31eb0eae762bbe5e2e960edf4494069598  **Field-error**  was removed and it was replaced by error as an element which is the one breaking the test that validates phoneNumberNotifications currently

Reply:
Actually to make it more functional we need to add another validation to validate for **error** to make it more generic